checks/prgenv/environ_check.py

```diff
@@ -13,12 +13,18 @@ class DefaultPrgEnvCheck(rfm.RunOnlyRegressionTest):
     descr = 'Ensure PrgEnv-cray is loaded by default'
     valid_prog_environs = ['builtin']
     valid_systems = ['daint:login', 'dom:login',
-                     'eiger:login', 'pilatus:login']
+                     'eiger:login', 'pilatus:login',
+                     'hohgant:login']
     executable = 'module'
     executable_opts = ['--terse', 'list']
     maintainers = ['TM', 'CB']
     tags = {'production', 'craype'}
 
+    @run_after('init')
+    def load_cray_module(self):
+        if self.current_system.name in ['hohgant']:
```

Comment:
why is it needed if it's already in the [config](https://github.com/eth-cscs/cscs-reframe-tests/blob/9e87e2ee9108d5d56239d92ec360f66d4414296e/config/cscs.py#L929) ?

Reply:
Because this test is running on the login environment, I thought it was better to not add a module for the `login` env, what do you think? We could also create a new env called `cray` but there not too many tests so I don't know if it's worth it at this point.